engines/sci/sound/drivers/fmtowns.cpp

```diff
@@ -461,12 +461,12 @@ int MidiDriver_FMTowns::open() {
 	return 0;
 }
 
-void MidiDriver_FMTowns::loadInstruments(const uint8 *data) {
-	if (data) {
-		data += 6;
-		for (int i = 0; i < 128; i++) {
-			_intf->callback(5, 0, i, data);
-			data += 48;
+void MidiDriver_FMTowns::loadInstruments(const SciSpan<const uint8> &data) {
+	if (data.size()) {
+		SciSpan<const uint8>::const_iterator it = data.cbegin() + 6;
+		for (int i = 0; i < 128; i++, it += 48) {
+			// TODO: Unsafe use of va_args
```

Comment:
This TODO is confusing. By `unsafe use`, do you mean the pointer is used without bounds checking (in `intf_loadInstrument` I guess?), or is there actual unsafe use of va_args itself somewhere?